packages/@lwc/engine/src/framework/api.ts

```diff
@@ -77,7 +75,7 @@ export interface CustomElementCompilerData extends ElementCompilerData {
 }
 
 export interface RenderAPI {
-    s(slotName: string, data: ElementCompilerData, children: VNodes, slotset: SlotSet): VNode;
```

Comment:
`api.s` was special because we need to determine what children to you for that vnode based on the allocation.... this is not longer needed. a `slot` element is just a regular html element and its vnode should be created via `api.h()` as any other element.

Reply:
This change is pending to be carry on the compiler.